app/src/main/java/com/babylon/wallet/android/domain/model/SecurityProblem.kt

```diff
@@ -0,0 +1,31 @@
+package com.babylon.wallet.android.domain.model
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.babylon.wallet.android.R
+
+sealed interface SecurityProblem {
+    data class EntitiesNeedBackup(
```

Comment:
why `EntitiesNeedBackup` here? Isn't this about not writing seed phrase down? 
So a name like "EntitiesNotRecoverable" would fit better in this case?

Reply:
because you need to back up entities seed phrase - thus entities need back up